cognitedata/reveal
# feat(react-components): add HOC to store Slicer state in Url

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/UX-1351

## Description :pencil:
This HOC when mounted, slicer state will be stored in the Url when the slider value changes

## How has this been tested? :mag:

In Storybook

## Test instructions :information_source:

- `cd react-components && yarn && yarn build && yarn storybook`
- Select `Reveal3DResources` example
- Open the canvas of the story in a new window
- Change the slider value in `Slicer Button` and confirm that the state changes in Url
- Can copy the Url and load it in new tab.

## Checklist :ballot_box_with_check:
